server/datastore/mysql/software.go

```diff
@@ -895,3 +915,82 @@ ORDER BY
 	}
 	return hosts, nil
 }
+
+func (ds *Datastore) InsertVulnerabilities(
+	ctx context.Context,
+	vulns []fleet.SoftwareVulnerability,
+	source fleet.VulnerabilitySource,
+) (int64, error) {
```

Comment:
Let's document what does the returned `int64` means. (what's the value in case of inserting existing entries?)